### ISSUE TITLE
[Accessibility] Error message truncated on General tab

### ISSUE THREAD

**Issue by shahparth-lab** [Azure/LogicAppsUX#3255]
### Describe the Bug with repro steps

User Experience: 
User would face difficulty to see all the controls after setting the pixels up to 320 x 256 if the controls on screen are not clearly visible. 

Note: Login Credentials must not be provided within the bug.

Pre - Requisite:
Open Logic Apps Custom Connector Deployment General tab >> Open developers tool through F12 >> Open Developer's tool in separate window  >> Zoom the browser >> Adjust the pixels up to 320 X 256 and verify the issue.

1. Open https://ms.portal.azure.com and login via valid credentials.
2. Navigate to the Left Nav bar and select 'More Services', search 'Logic App Custom Connector' and Select any already created ''Logic App Custom Connector'
3. Press 'Edit' button on the menu bar on top of overview blade.
4. Navigate all the controls on the General tab
5. Input valid and invalid values on the General tab and Press Continue Button. Verify the errors on the page.
6. Press Update Connector. Verify the errors that appear on the page
7. Verify this issue.

Actual Result:
After setting the pixels up to 320 x 256, The host error message is getting truncate available on the General Tab.

Expected Result:
After setting the pixels up to 320 x 256,The host error message should not truncate available on the General Tab. the Host Error message should visible properly.
Note:
Same issue exists with 'Create an Integration account' text present in 'Create an Integration account' page.

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Browser

Chrome Version 96.0.4664.45 (Official Build) (64-bit)

AB#25072637

**Comment by hartra344** [comment]
@shahparth-lab This should be on the Power Automate team. We do not own this UX.
